src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -526,7 +526,7 @@ def test_create_and_delete_advanced_sequence_bad_type(session):
 @pytest.mark.resource_name("4190/0")
 @pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
 def test_create_and_delete_advanced_sequence_attribute_enum_with_converter(session):
-    properties_used = ['lcr_impedance_auto_range']
+    properties_used = ['voltage_level']
```

Comment:
@DavidCurtiss This was the only change I had to make in the system tests to get them all to pass for nidcpower. The error I get with the original attribute is: `nidcpower.errors.DriverError: -1074118544: The specified attribute is not supported in an advanced sequence for this device.` I'm not sure why it doesn't work anymore, but it seems like which attribute is used doesn't matter for this test?

Reply:
Does that error happen only over gRPC, or via the local tests, too?

Reply:
This is fishy, need to get to the bottom of it.

Reply:
See #1839 for what I believe is the proper fix.

Reply:
@ni-jfitzger what is keeping that fix (1839, test removal) from going in?

Reply:
We're just waiting for @marcoskirsch to review and confirm that our conclusions are reasonable.